extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToPage.java

```diff
@@ -105,13 +106,18 @@ default ToPage<DictionaryKeys, long[]> getDictionaryKeysToPage() {
     }
 
     /**
-     * @return an reverse lookup map of the dictionary.
+     * @return a reverse lookup map of the dictionary.
      * @apiNote null iff {@link #getDictionaryChunk()} is null.
      */
     default LongBitmapStringSet.ReversibleLookup getReversibleLookup() {
         return null;
     }
 
+    /**
+     * @return the factory to create the materializers for this column.
+     */
+    PageMaterializerFactory getPageMaterializerFactory();
```

Comment:
It's not immediately obvious to me that this belongs as _part_ of `ToPage`. No doubt it's convenient (and, we might argue is in the same style as the other "helpers" `ToPage#getChunkType`, `ToPage#getNativeType`). Generally, I'm trying to minimize the expansion of "public" interfaces. That said, this is already an implementation exposing interface, and I don't think this additional exposure is egregious in any way; just that I wish we were better at having a separation between public interfaces and implementation details. Oh well. (I don't think you need to necessarily make any changes here, just venting a bit.)